### ISSUE TITLE
Single Box Shadow is buggy with the new version

### ISSUE THREAD

**Issue by emin93** [Compass/compass#1809]
On Page: http://compass-style.org/examples/compass/css3/box_shadow/

I was looking for:
Single Box Shadow

But instead I found:
Just a 5px border bottom on the first example.

**Comment by eddie-ruva** [comment]
@emin93 In the first example the single-box-shadow mixin is using the default values which are 

``` css
0px 5px #333333
```

From the w3c spec: 

**1st length:** Specifies the horizontal offset of the shadow. A positive value draws a shadow that is offset to the right of the box, a negative length to the left.

**2nd length:** Specifies the vertical offset of the shadow. A positive value offsets the shadow down, a negative one up.

This is why it looks like a 5px border bottom. 

If I understood your description right I think there is no issue with the output.

**Comment by emin93** [comment]
Thanks for your fast reply. Let me try to explain again. In the older version of Compass, single-box-shadow had other default values than in the new one. In the new version of Compass, the default values are chosen impractical, because it looks like a 5px border bottom.

**Comment by eddie-ruva** [comment]
@emin93 Sorry for not replying back for some reason I did not see your comment before. I get what you mean now thanks for explanation. I looked up the code for an older version (0.12.0) and it seems like the default values have not been changed. You can see the file that I looked at [here](https://github.com/Compass/compass/blob/v0.12.0/frameworks/compass/stylesheets/compass/css3/_box-shadow.scss). If you remember the old values maybe I can create a pull request to bring them back so they are backwards compatible.

**Comment by emin93** [comment]
@eddie-ruva I'm sorry, that it took me so long to respond. I've got some old websites running with 0.12.6. Compare the [latest stable version](https://github.com/Compass/compass/blob/stable/core/stylesheets/compass/css3/_box-shadow.scss) with the [version you sent me](https://github.com/Compass/compass/blob/v0.12.0/frameworks/compass/stylesheets/compass/css3/_box-shadow.scss) and you'll see at line 48, it seems that some of the default values for the single-box-shadow are not set. My old websites default values are:
`box-shadow: 0px 0px 5px #333333;`

The new default values are:
`box-shadow: 0px 5px #333333;`

**Comment by dfunckt** [comment]
@eddie-ruva, yes, it is buggy. See [line 69 in _box-shadow.scss](https://github.com/Compass/compass/blob/stable/core/stylesheets/compass/css3/_box-shadow.scss#L69). It should be `$voff`. There's an unmerged PR about that: #1806 

@emin93 you may use `@include box-shadow()` until this is fixed.
